index.js

```diff
@@ -119,13 +121,31 @@ module.exports = {
       allowObject: true,
     }],
 
+    // This rule is problematic in JSX
+    // https://eslint.org/docs/rules/spaced-comment
+    'spaced-comment': 'off',
+
+    // This is better managed by other rules
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/function-component-definition.md
+    'react/function-component-definition': 'off',
+
+    // It is common practice to use .js for React
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-filename-extension.md
+    'react/jsx-filename-extension': 'off',
+
+    // AirBnB uses 'error' but it's too disruptive during development
+    // and we shouldn't prevent compilation because of it
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unused-prop-types.md
+    'react/no-unused-prop-types': ['warn'],
```

Comment:
I have the same thoughts here [as with setting `no-unused-vars` to warn mode](https://github.com/americanexpress/eslint-config-amex/pull/25#discussion_r421861596), that is keeping at "error" mode is the pit of success (avoid buildup of unhelpful code) and there's a way to get to advanced mode (`--no-verify`)

Reply:
```suggestion
```